### ISSUE TITLE
Generate man-pages and bash-completions when building

### ISSUE THREAD

**Issue by karlowich** [OpenMPDK/xNVMe#363]
This is to avoid manual generation of man-pages and completions and to avoid storing autogenerated stuff in the repository

**Comment by karlowich** [comment]
@safl If we add this to the build process we also get a build-time dependency for `txt2man` I don't think we want that.

**Comment by safl** [comment]
I agree. Although others might argue that anything generated should never be committed... however... I do prefer reducing build-dependencies on the project by having all codegen done ahead of time, such that building from source-archive has only the *actual* build and runtime dependencies, not the vast myriad of dependencies introduced by codegen.
